src/backend/interfaces/REST/go.server/middlewares/authentication.go

```diff
@@ -25,9 +28,14 @@ func BasicAuthFromCache(cache *redis.Client, realm string) gin.HandlerFunc {
 		// Get provided auth headers
 		var user_id, access_token string
 		var ok bool
-		if user_id, access_token, ok = c.Request.BasicAuth(); !ok {
-			kickUnauthorizedRequest(c, realm)
+
+		//Try auth-scheme 'Bearer' then 'Basic'
+		if user_id, access_token, ok = BearerAuth(c.Request); !ok {
+			if user_id, access_token, ok = c.Request.BasicAuth(); !ok {
```

Comment:
both Bearer and Basic authentication schema must be acccepted by API.

Basic is cool for testing purposes, Bearer is what we must use for real condition (so Front use Bearer as we discussed previously)